eth_portfolio/portfolio.py

```diff
@@ -29,41 +44,137 @@
 
 class PortfolioWallets(Iterable[PortfolioAddress], Dict[Address, PortfolioAddress]):  # type: ignore [misc]
     """
-    A container that holds all `PortfolioAddress` objects for a specific `Portfolio`.
+    A container that holds all :class:`PortfolioAddress` objects for a specific :class:`Portfolio`.
+
     Works like a `Dict[Address, PortfolioAddress]` except when you iterate you get the values instead of the keys.
-    You should not initialize these. They are created automagically during Portfolio init.
+    You should not initialize these. They are created automagically during :class:`Portfolio` init.
+
+    :ivar _wallets: A checksummed dictionary of PortfolioAddress objects.
```

Comment:
move the _wallets info to the line immediately below the assignment of the self._wallets var